### ISSUE TITLE
moduleMovedForSplit messages refer to old project URLs

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2456]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)| @glyph reported|
|-|-|
|Trac ID|trac#2456|
|Type|defect|
|Created|2007-02-17 09:26:25Z|
|Branch|https://github.com/twisted/twisted/tree/old-project-urls-2456-2|

For example,

  http://twistedmatrix.com/documents/current/api/twisted.protocols.http.html

points to

  http://twistedmatrix.com/projects/web

whereas it _should_ point to

  http://twistedmatrix.com/trac/wiki/TwistedWeb

Attachments:

* [old-project-urls-2456.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/d56/d56d196e44d15d5866067c456ab63ae19a677075/d5c4d872f32364928d425591067f521f0f91dbed.patch) (1456 bytes) - added by thijs on 2009-03-22 16:29:50Z - adding patch that ill put in a branch and run against docbuilder when svn is back up

<details><summary>Searchable metadata</summary>

```
trac-id__2456 2456
type__defect defect
reporter__glyph glyph
priority__normal normal
milestone__ 
branch__branches_old_project_urls_2456_2 branches/old-project-urls-2456-2
branch_author__thijs thijs
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1171704385000000 1171704385000000
changetime__1237740322000000 1237740322000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra commented|
|-|-|

(In [25336]) Branching to 'old-project-urls-2456'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

Fixes from r25337 up for review.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_4"></a>|@therve set owner to @thijstriemstra|
|-|-|

The changes in http.py and glossary.xhtml are wrong: there are missing the wiki/ part.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra removed owner|
|-|-|

Thanks for the review, fixed it in r25353

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_6"></a>|@radix set owner to @thijstriemstra|
|-|-|

Just one comment: Twisted Reality doesn't even exist as a project any more, so the line in glossary.xhtml is a bit misleading:

"Twisted Reality has since been broken off into a separate project."

The line should either be removed, or replaced by:

"Twisted Reality has been superseded by the <a href="http://www.divmod.org/trac/wiki/DivmodImaginary">Imaginary</a> project"

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_7"></a>|@thijstriemstra removed owner|
|-|-|

Added the link in r25369, up for review.

**Comment by twisted-trac** [comment]
|[<img alt="mwhudson's avatar" src="https://avatars.githubusercontent.com/u/672022?s=50" width="50" height="50">](https://github.com/mwhudson)<a name="note_8"></a>|@mwhudson set owner to @thijstriemstra|
|-|-|

I think the call in twisted.im needs updating too.  Otherwise it's fine.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_10"></a>|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

Replying to [mwh](#note_8):
> I think the call in twisted.im needs updating too.  Otherwise it's fine.

Fixed url in r26478. Also see [#3581](https://github.com/twisted/twisted/issues/3581) (Remove all the pre-split compatibility code).

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_11"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

The pre-split stuff will conflict with this.  Probably it'd be simplest to just drop it from this branch.  Another issue is that faq.xhtml no longer exists in trunk.  Are the changes which were made to that FAQ in this branch already on the website FAQ page?  If so, those changes can be dropped too (otherwise they should be moved there).

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_12"></a>|@thijstriemstra commented|
|-|-|

(In [26499]) Branching to 'old-project-urls-2456-2'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_13"></a>|@thijstriemstra set status to `assigned`|
|-|-|

Applied the [[old-project-urls-2456.patch](None/old-project-urls-2456.patch) patch] to the new branch and the docbuilder is [happy](http://buildbot.twistedmatrix.com/builders/documentation/builds/356) so merging it.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_14"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [26502]) Merge old-project-urls-2456-2: the core howto glossary referred to some old project urls.

Author: thijs
Reviewer: therve, radix, mwh, exarkun
Fixes: [#2456](https://github.com/twisted/twisted/issues/2456)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
